config/opal_setup_component_package.m4

```diff
@@ -1,105 +0,0 @@
-dnl -*- shell-script -*-
-dnl
-dnl Copyright (c) 2004-2005 The Trustees of Indiana University and Indiana
-dnl                         University Research and Technology
```

Comment:
This file looks like it was removed because it is unused.  Yay!

But it should probably be a separate commit, just to show that it was removed because it's dead code -- not part of the revamp to go from 10 m4 params to 9 m4 params.

Reply:
Sure, wil do